eda/setup.py

```diff
@@ -20,16 +20,15 @@
 submodule = 'eda'
 install_requires = [
     # version ranges added in ag.get_dependency_version_ranges()
-    'numpy',
-    'scipy',
-    'scikit-learn',
-    'pandas',
-    'matplotlib>3.3',
+    'numpy',  # version range defined in `core/_setup_utils.py`
+    'scipy',  # version range defined in `core/_setup_utils.py`
+    'scikit-learn',  # version range defined in `core/_setup_utils.py`
+    'pandas',  # version range defined in `core/_setup_utils.py`
+    'matplotlib>3.3,<3.7',
     'missingno>=0.5.1,<0.6',
     'phik>=0.12.2,<0.13',
     'seaborn>=0.12.0,<0.13',
-    'ipython>=7.34,<9.0',
```

Comment:
can you explain why you removed ipython?

Reply:
`ipywidget` dependency already have a transitive dependency to `ipython`

Reply:
Gotcha, thanks!